task/common/identifier.go

```diff
@@ -12,38 +12,52 @@ import (
 	"github.com/aohorodnyk/uid"
 )
 
-type Identifier string
+type Identifier struct {
```

Comment:
Ah, this could use a comment too.

Reply:
I'm afraid I'll leave it for the next time. 😅